qdrant-landing/content/articles/vector-search-resource-optimization.md

```diff
@@ -0,0 +1,576 @@
+
+#### Benefits of Scalar Quantization:
+
+| Benefit                          | Description                                                                                                      |
+|----------------------------------|------------------------------------------------------------------------------------------------------------------|
+| **Memory usage will drop**       | Compression cuts memory usage by a factor of 4. Qdrant compresses 32-bit floating-point values (float32) into 8-bit unsigned integers (uint8). |
+| **Accuracy loss is minimal**     | Converting from float32 to uint8 introduces a small loss in precision. Typical error rates remain below 1%, making this method highly efficient. |
+| **Best for specific use cases**  | To be used with high-dimensional vectors where minor accuracy losses are acceptable. |
+
+#### Set it up as you create the collection:
+
+```python
+client.create_collection(
+   collection_name="{collection_name}",
+   vectors_config=models.VectorParams(size=1536, distance=models.Distance.COSINE),
+   quantization_config=models.ScalarQuantization(
+       scalar=models.ScalarQuantizationConfig(
+           type=models.ScalarType.INT8,
+           quantile=0.99,
+           always_ram=True,
+       ),
+   ),
+)
+```
+When working with Qdrant, you can fine-tune the quantization configuration to optimize precision, memory usage, and performance. Here’s what the key configuration options include:
+
+| Configuration Option | Description                                                                                                      |
+|----------------------|------------------------------------------------------------------------------------------------------------------|
+| `type`             | Specifies the quantized vector type (currently supports only int8).                                              |
+| `quantile`         | Sets bounds for quantization, excluding outliers. For example, 0.99 excludes the top 1% of extreme values to maintain better accuracy. |
+| `always_ram `      | Keeps quantized vectors in RAM to speed up searches.                                                             |
+
+Adjust these settings to strike the right balance between precision and efficiency for your specific workload.
+
+---
+Learn More about [**Scalar Quantization**](/documentation/guides/quantization/)
+
+---
+
+#### Binary Quantization
+
+**Binary quantization** takes scalar quantization to the next level by compressing each vector component into just **a single bit**. This method achieves unparalleled memory efficiency and query speed, reducing memory usage by a factor of 32 and enabling searches up to 40x faster.
+
+#### **Benefits of Binary Quantization:**
+
+Binary quantization is ideal for high-dimensional datasets and compatible embedding models, where compression and speed are paramount. 
```

Comment:
```suggestion
Binary quantization is ideal for large-scale datasets and compatible embedding models, where compression and speed are paramount.
```

The datasets are not highly-dimensional themselves, but BQ works better for high-dimensional vectors. However, it should work best when the embedding dimensions are centered around zero, but that's not that important here.